packages/browser/src/eventbuilder.ts

```diff
@@ -74,7 +74,8 @@ export function eventFromPlainObject(
   if (syntheticException) {
     const frames = parseStackFrames(syntheticException);
     if (frames.length) {
-      event.stacktrace = { frames };
+      // event.exception.values[0] has been set above
+      (event.exception as { values: Exception[] }).values[0].stacktrace = { frames };
```

Comment:
We need to update this for `exceptionFromError` as well right?

Reply:
Where do you mean?

`exceptionFromError` currently has this which is correct:
```ts
  if (frames.length) {
    exception.stacktrace = { frames };
  }
```

Reply:
oh man I misread 🤦 wooooops